### ISSUE TITLE
Plugin loads and initializes but does not run

### ISSUE THREAD

**Issue by mdark2385** [bryanbartow/homebridge-alarm.com#4]
Any suggestions on this issue? No "nag" screens on the website and no errors reported. Website not showing that any login ever occurred.

**Comment by bmartin5692** [comment]
Hi @mdark2385. Would you be able to provide your homebridge.log and .err files? They should be located at /var/log/homebridge.log and /var/log/homebridge.err.

Do you have any other plugins configured in your homebridge yet that are working?

**Comment by bryanbartow** [comment]
@mdark2385 As @bmartin5692 suggested, I'd probably need to see some logs before I knew where to begin. FWIW, you are the first to report this particular issue.

**Comment by mdark2385** [comment]
I'm running homebridge on a mac. Note sure where to find the log files. I've been searching and can't seem to locate them. Also, the Wink and Nest plugins run and work fine. The alarmdotcom accessory did work at one point - would login to the website and get the status (although could not set it). Now, the plugin loads and initializes when I start running home bridge but does nothing after that point.

**Comment by nmcdan75** [comment]
@mdark2385 - how difficult was it to get an API key and question from Wink?

**Comment by bryanbartow** [comment]
@mdark2385 Which version of the plugin are you using? Also, @bmartin5692 pointed out the location of the log files.

**Comment by mdark2385** [comment]
I'm running version 0.1.8. I'm still attempting to locate the log files, however they are not at  /var/log/homebridge.log and /var/log/homebridge.err. Any ideas on if they are located in a different location on a mac? I can't seem to find them anywhere...

**Comment by bryanbartow** [comment]
@mdark2385 You can also look in `~/.homebridge` directory.

**Comment by mdark2385** [comment]
The only files in the ~/.homebridge directory are the config file and the persist folder.  Still cannot locate any log files.

here is a copy of home bridge startup:

Mac-mini:~ xxx$ homebridge
Loaded plugin: homebridge-alarmdotcom

## Registering accessory 'homebridge-alarmdotcom.Alarmdotcom'

Loaded plugin: homebridge-nest

## Registering platform 'homebridge-nest.Nest'

Loaded plugin: homebridge-wink

## Registering platform 'homebridge-wink.Wink'

## Loaded config.json with 1 accessories and 1 platforms.

Loading 1 platforms...
[Wink] Initializing Wink platform...
[Wink] Fetching Wink devices.
Loading 1 accessories...
[Security Panel] Initializing Alarmdotcom accessory...
[Wink] Device Added - Group thermostats, ID 105377, Name xxx Hallway Thermostat (Nest)
[Wink] Device Ignored Not In HomeKit - Group hubs, ID 339696, Name Hub
[Wink] Device Added - Group light_bulbs, ID 1312846, Name Bedroom
[Wink] Device Added - Group light_bulbs, ID 1312863, Name Living Room
[Wink] Device Ignored Not In HomeKit - Group hubs, ID 339696, Name Living Room Remote
[Wink] Device Ignored Not In HomeKit - Group hubs, ID 339696, Name Bedroom Remote
[Wink] Initializing platform accessory 'xxx Thermostat (Nest)'...
[Wink] Initializing platform accessory 'Bedroom'...
[Wink] Initializing platform accessory 'Living Room'...
Scan this code with your HomeKit App on your iOS device to pair with Homebridge:

```
┌────────────┐     
│ 023-23-023 │     
└────────────┘     
```

Homebridge is running on port 51826.

**Comment by bryanbartow** [comment]
@mdark2385 This may be a dumb question, but what exactly is causing you to think the plugin is not running? Have you tried setting up scenes and executing them via Siri or an app?

**Comment by mdark2385** [comment]
The plugin did work for me at one point. The terminal window would show each time the plugin connected to the alarm.com website as well as the website show a login had occured. I'm using the Insteon App. A security panel shows up with home bridge but it does not have any information (does not get a status or allow me to change it.) I tried installing home bridge on my other mac and the same thing is occurring. I've tried running home bridge as a superuser thinking that maybe some permissions were the issue but that has no effect.

**Comment by nmcdan75** [comment]
@bryanbartow and @mdark2385 I too am finding the alarm.com plugin is not working on my homebridge anymore. It will show up as an accessory, but throws up various error messages when looking at it from any of the apps I have to manage HomeKit.

**Comment by bryanbartow** [comment]
@mdark2385 @nmcdan75 Which version of home bridge are you each running? It's working fine for me running the latest alarm.com plugin with the latest version of home bridge. I'm using the MyTouchHome app and I can change state, although the change takes a few second from the time I tap the button.

**Comment by mdark2385** [comment]
homebridge version 0.2.16
homebridge-alarmdotcom version 0.1.8

**Comment by bryanbartow** [comment]
@mdark2385 Hmm...Interesting. Same here and it works. Tough to say, really. Are you getting specific error messages from the HomeKit management apps you're using?

**Comment by mdark2385** [comment]
Ok, so I tried multiple home kit apps since @bryanbartow mentioned using MyTouchHome.  Different apps are behaving differently but it appears I can control the security system now. The Insteon+ app will not work to control the security panel. I tried a few other free home kit apps and they are incorrectly reporting the state of the system but are allowing be to see the state. I'm going to do some more testing with different apps but the plugin seems to be working now.
